curriculum/challenges/english/25-front-end-development/lecture-understanding-the-event-object-and-event-delegation/67338e93b75977a344cf6d40.md

```diff
@@ -10,6 +10,122 @@ dashedName: how-do-event-bubbling-and-event-delegation-work
 
 Watch the lecture video and answer the questions below.
```

Comment:
There is a small issue with the video.
Waiting for it to be fixed, then I will update the video id in this PR